doc/user_guide/lfric.rst

```diff
@@ -3242,6 +3254,9 @@ pseudo-random numbers in the interval ``0 <= x < 1``::
 where ``RAND()`` is some function that returns a new pseudo-random number
 each time it is called.
 
+Due to its simplistic implementation, this kernel does *not* support
+redundant computation and hence has ``OPERATES_ON=owned_dof``.
```

Comment:
Due to different parallel elements using independent random streams, this built-in has ``OPERATES_ON=owned_dof``, which will prevent optimisations such as redundant computation and computed annexed dofs involving this built-in.